clients/ops/admin-ui/src/features/datastore-connections/add-connection/manual/DSRCustomization.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, Center, Spinner, VStack } from "@fidesui/react";
+import { useAppSelector } from "app/hooks";
+import { useAlert, useAPIHelper } from "common/hooks";
+import { selectConnectionTypeState } from "connection-type/connection-type.slice";
+import {
+  useCreateAccessManualWebhookMutation,
+  useGetAccessManualHookQuery,
+  usePatchAccessManualWebhookMutation,
+} from "datastore-connections/datastore-connection.slice";
+import {
+  CreateAccessManualWebhookRequest,
+  CreateAccessManualWebhookResponse,
+  PatchAccessManualWebhookRequest,
+  PatchAccessManualWebhookResponse,
+} from "datastore-connections/types";
+import React, { useEffect, useRef, useState } from "react";
+
+import DSRCustomizationForm from "./DSRCustomizationForm";
+import { Field } from "./types";
+
+const DSRCustomization: React.FC = () => {
+  const mounted = useRef(false);
+  const { successAlert } = useAlert();
+  const { handleError } = useAPIHelper();
+  const [isSubmitting, setIsSubmitting] = useState(false);
+  const [fields, setFields] = useState([] as Field[]);
+
+  const { connection } = useAppSelector(selectConnectionTypeState);
+
+  const { data, isFetching, isLoading, isSuccess } =
+    useGetAccessManualHookQuery(connection!.key);
```

Comment:
This GET request is called on the first "create" too which will be a 404 since it doesn't exist yet. Is that ok?

Reply:
The reason for the GET request on the initial page load was to let me know if the given connection key has any existing fields. This way I know whether a POST or PATCH API endpoint needs to be invoked when the user clicks the Save button.